docs/hackathon/startup-guide.md

```diff
@@ -1,64 +1,70 @@
 ---
 id: startup-guide
-title: Hackathon Startup Guide (10min)
-sidebar_label: Hackathon Startup Guide (10min)
+title: Hackathon Startup Guide
+sidebar_label: Hackathon Startup Guide
 ---
 
-## Introduction to NEAR
+## Welcome to NEAR
 
-First thing you need to do is go over some of [the Smart Contract Basics](../quick-start/blockchain-prerequisite). You don't need to become an expert, just get acquainted at this point.
+You might have come to this site because you are 
+1. In a room, packed with great minds, who are eager to develop some amazing applications in the next few hours, days, or month;
+2. A student, looking for a badass CS project;
+3. Really curious on getting started - fast. 
 
-Then head over to the [Beginner Quickstart](../quick-start/near-studio-ide.md). That will get you actually exploring the code.
+If you want to get started developing on NEAR you are at the right place. Let's kick things off!
 
-I recommend getting familiar with how to actually write code in a smart contract in studio, and hitting the `run` button. That will give you a good foundation for local development.
+## Why are we building NEAR?
 
-For the start of the hackathon, it's fine to do development in the studio, but eventually you're going to want to set up a Github repo and share code with your team. That will be easier to do with a local setup.
+You might have heard of distributed computing, databases, or computer networks; all of which play a role in blockchains. 
 
-This is the quick and dirty way to do that:
+Currently, most web-services utilise a single server and a single database to process your request and provide information. This infrastructure is usually managed by an individual entity, who is treating all of their data processing like a black box. Meaning, the request goes in, something happens, and the user receives an output. 
 
-## Running official TestNet locally
+While the company may make claims, and rely on third-parties to verify those, the user will never be able to verify what happened in the black box. This system relies on trust between users and companies.
 
-By default we use Docker to run the client \(with auto-updating via watchtower to upgrade the node to the new versions automatically\).
+Near is basically building the infrastructure for a new Internet which makes it harder for giant companies to steal your data and for bad guy countries to shut it down. People have been trying to figure this stuff out with similar technologies since 2008 but it’s been slow going.
```

Comment:
the tone here is a little casual. specifically: _"harder for giant companies to steal your data and for bad guy countries to shut it down"_.  not sure if this makes the docs more relatable or not

Reply:
I will change it and provide a relatable example